### ISSUE TITLE
Inconsistencies between rates and staff pages

### ISSUE THREAD

**Issue by paulodamaso** [zerocracy/farm#1827]
Staff and rates pages have inconsistencies between them: there are members in staff which does not appear in rates page. Please see `pm/cost/rates` and `pm/staff/roles`.

**Comment by 0crat** [comment]
@g4s8[/z](https://www.0crat.com/u/g4s8) please, pay attention to this issue

**Comment by g4s8** [comment]
@paulodamaso why do you think it's a bug? Project may have some people without rate, e.g. `PO` doesn't have rate but has a role.

**Comment by paulodamaso** [comment]
@g4s8 Right, `PO` is a true case here. Well, maybe we should put some warning in `rates` page telling that maybe not all team members are listed there due these situations or show staff members with no rate or $0 rate in this page as well, WDYT?

**Comment by g4s8** [comment]
@paulodamaso I don't think it's a valid bug, because this warning will be shown always: each project has a PO, so in every project this situation happens. Also rates page doesn't claim that it contains all project members.

**Comment by paulodamaso** [comment]
@g4s8 Okay, it was just a suggestion to improve  these pages because I've found it a bit confusing at first.
